python/servo/command_base.py

```diff
@@ -531,6 +533,7 @@ def build_env(self, is_build=False):
         # Work around https://github.com/servo/servo/issues/24446
         # Argument-less str.split normalizes leading, trailing, and double spaces
         env['RUSTFLAGS'] = " ".join(env['RUSTFLAGS'].split())
+        print(env['RUSTFLAGS'])
```

Comment:
Was this a temporary print statement added for debugging? If not, can we print this with a suffix like `RUSTFLAGS=`

Reply:
:innocent: 

Yeah, this was left over. I've removed it now.